ai-platform-unified/notebooks/official/explainable_ai/gapic-custom_image_classification_online_explain.ipynb

```diff
@@ -31,7 +31,7 @@
     "repo": "snippets_housekeeping.ipynb"
```

Comment:
<p>Please incorporate the docs links for setting machine types:</p><p> </p><p>https://cloud.google.com/vertex-ai/docs/training/configure-compute</p><p>https://cloud.google.com/vertex-ai/docs/predictions/configure-compute</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>done</p>